compiler/rustc_ast/src/ast.rs

```diff
@@ -1759,6 +1762,11 @@ pub enum CaptureBy {
     },
     /// `move` keyword was not specified.
     Ref,
+    /// `use |x| y + x`.
```

Comment:
```suggestion
    /// `use |x| y + x`.
    ///
    /// Note that if you have a regular closure like `|| x.use`, this will *not* result
    /// in a `Use` capture. Instead, the `ExprUseVisitor` will look at the type
    /// of `x` and treat `x.use` as either a copy/clone/move as appropriate.
```